cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/ConnectorMockTest.java

```diff
@@ -277,18 +277,17 @@ public void setup(VertxTestContext testContext) throws InterruptedException {
         kafkaConnectOperator.createWatch(NAMESPACE, e -> testContext.failNow(e))
             .onComplete(testContext.succeeding())
             .compose(watch -> {
-                kafkaConnectS2iOperator = new KafkaConnectS2IAssemblyOperator(vertx, pfa, ros, config, x -> api);
+                kafkaConnectS2iOperator = new KafkaConnectS2IAssemblyOperator(vertx,
+                    pfa,
+                    ros,
+                    config,
+                    x -> api);
```

Comment:
Can we rename `x` to something a little more descriptive?

Reply:
Not actually my code but sure.